src/main/java/nl/thehyve/ocdu/services/OpenClinicaService.java

```diff
@@ -30,14 +32,14 @@
 @Service
 public class OpenClinicaService {
 
-    SOAPRequestFactory responseFactory = new SOAPRequestFactory();
+    SOAPRequestFactory requestFactory = new SOAPRequestFactory();
```

Comment:
Okay. I do not what you had to fix so I don't know the full context but it would seem that the rebase was one commit to far. you can also use commit hash (hash displayed in git log entries for instance) as a target of rebasing as well when you are not sure. When we merge this PR this commit should not appear twice so it shouldn't be a problem, but lets take care of stuff like this better in future PRs.